lib/elixir/lib/kernel.ex

```diff
@@ -2005,6 +2005,10 @@ defmodule Kernel do
       inspect(fn a, b -> a + b end)
       #=> #Function<...>
 
+  The `Inspect` protocol can be derived to hide certain fields
```

Comment:
I have a lot of doubts regarding what to document here (`Kernel.inspect/2`). The documentation seems to be more focused about the usage and expected results of `inspect/2`, and not about the implementation of custom inspects 😅 

I thought that mentioning this new functionality, and a link would be enough :)